conftrak/test/conftest.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+from conftrak.ignition import start_server
+from doct import Document
+from mock import patch
+import ujson
+import jsonschema
+import uuid
+import sys
+import time as ttime
+import subprocess
+import contextlib
+from conftrak.client.commands import ConfigurationReference
+from conftrak.exceptions import ConfTrakNotFoundException
+from conftrak.ignition import Application
```

Comment:
Do we need all of these imports?

Reply:
no - removed in https://github.com/NSLS-II/conftrak/pull/7/commits/9d38ebc5017d928194c606b8ebafdac23580965d and https://github.com/NSLS-II/conftrak/pull/7/commits/0a330b914bb7b086592c1e3d8288ba1f010b5456